### ISSUE TITLE
What is the future of SOUL?

### ISSUE THREAD

**Issue by kyr0** [soul-lang/SOUL#75]
Hi :)

just asking as the last release was about Feb and last commit in April?

Best,
Aron

**Comment by ZonderP** [comment]
Yeah, alarmingly silent here...
No comments from the Devs on issues posted since July and also otherwise no signs of activity since quite a long time.
I'm wondering too about what's happening to this promising project.

**Comment by kunitoki** [comment]
It's dead. ROLI bankrupt, JUCE sold to PACE and Jules back at Tracktion. I don't see anything bright for the future of SOUL

**Comment by synxroform** [comment]
I hope it will become community-driven project. Of course it's possible to go back to C, but you will lose JIT.

**Comment by kunitoki** [comment]
Unluckily it's closed source without any plan in making it open, besides the language specifications

**Comment by ceelian** [comment]
That's a pity not only for the founders but also for the devs that used SOUL.

Some devs were aware of this risk back then see [HN](https://news.ycombinator.com/item?id=25845002) post but afaik they didn't open source and relicense it.



![Screenshot_20220209-122554_Chrome](https://user-images.githubusercontent.com/116299/153190967-e3c44a1f-0807-486d-ae51-6b6b32385ef6.jpg)

**Comment by kunitoki** [comment]
Lol https://musictech.com/news/industry/roli-bankruptcy-luminary-reboot-lumi/

**Comment by 0xchase** [comment]
Yeah an update from the devs would be nice. I was originally thinking of using SOUL in a commercial product but have since switched to Rust due to the silence here.

**Comment by dathinaios** [comment]
@0xchase I agree, even if it just a reassurance without details. They had indicated that there was going to be an announcement in March and some reason that they can no talk about it so I am hopeful. Still frustrating though...

**Comment by thezhe** [comment]
Call me crazy, but I use SOUL in production and it works great. There's definitely issues with JIT and the lack of updates, but the language itself is great for DSP.

**Comment by nicolasdanet** [comment]
FYI: https://forum.juce.com/t/the-future-of-soul-ide/48162/36?u=nicolasdanet

**Comment by 0xchase** [comment]
More FYI: https://www.joinsoundwide.com/en/sound-stacks.html

**Comment by nicolasdanet** [comment]
FYI: https://youtu.be/FHtahCOBfgo?t=755

SOUL is dead, isn't it?

**Comment by shayneoneill** [comment]
Yup. Shame. The concept of "shaders for audio" ought be industry shaking, but alas, here we are. 

The ROLI we once knew is dead. The market aint got no soul. If you'll excuse the pun.
